static/src/stylesheets/module/content/_gallery.head.scss

```diff
@@ -1,22 +1,68 @@
 .content__labels--gallery,
-.l-side-margins--gallery .content__meta-container,
 .content__headline--gallery,
-.content__standfirst--gallery {
+.gallery__header {
     @include mq(desktop) {
         margin-left: gs-span(3) + $gs-gutter;
         max-width: gs-span(8);
     }
 }
 
-.l-side-margins--gallery {
-    .content__meta-container {
-        position: relative;
-        width: auto;
+.gallery__meta-container {
```

Comment:
Not 100% sure if the meta css should go in the head css, or not. Technically it isn't normally in the "first screen", but it could be. In those cases you see the page jumping slightly as non-head css is loaded.